v-next/hardhat/src/internal/builtin-plugins/solidity-test/index.ts

```diff
@@ -0,0 +1,14 @@
+import type { HardhatPlugin } from "../../../types/plugins.js";
+
+import { task } from "../../core/config.js";
+
+const hardhatPlugin: HardhatPlugin = {
+  id: "builtin:solidity-test",
+  tasks: [
+    task(["test:solidity"], "Run the Solidity tests")
```

Comment:
We should consider how we want this to look at the command line. Rather than block on something that is trivial to change, I chose `test:solidity`.